.github/workflows/team-sync.yml

```diff
@@ -0,0 +1,76 @@
+name: Teams
```

Comment:
Since we changed the name of the workflow, let me add a suggestion to change the *filename* accordingly to `teams.yml`. I wouldn't comment on it, if a later change wouldn't cause this to be a separate "workflow" in the Github Actions UI, which would result in two entries in https://github.com/NixOS/nixpkgs/actions with the same user-facing name. For that reason a later change is undesirable.